protocol/migrations/sqlite/1699554099_message_segments.up.sql

```diff
@@ -0,0 +1,11 @@
+CREATE TABLE IF NOT EXISTS message_segments (
```

Comment:
Also store the `SigPubKey` to prevent malicious peers from sending segments with invalid payloads, which would effectively invalidate the message reassembly process.